fineract-provider/src/main/java/org/apache/fineract/commands/service/CommandWrapperBuilder.java

```diff
@@ -3710,6 +3710,30 @@ public CommandWrapperBuilder loanPayOfLoan(Long loanId) {
         return this;
     }
 
+    public CommandWrapperBuilder updateHouseholdExpenses(long householdExpenseId) {
+        this.actionName = "UPDATE";
+        this.entityName = "HOUSEHOLD_EXPENSES";
+        this.entityId = householdExpenseId;
+        this.href = "/householdexpenses/" + householdExpenseId;
+        return this;
+    }
+
+    public CommandWrapperBuilder addHouseholdExpenses(long clientId) {
+        this.actionName = "ADD";
+        this.entityName = "HOUSEHOLD_EXPENSES";
+        this.clientId = clientId;
+        this.href = "/householdexpenses/" + clientId;
+        return this;
+    }
+
+    public CommandWrapperBuilder deletHouseholdExpenses(long householdExpenseId) {
+        this.actionName = "DELETE";
+        this.entityName = "HOUSEHOLD_EXPENSES";
+        this.entityId = householdExpenseId;
+        this.href = "/householdexpenses/" + clientId;
```

Comment:
Is this intentional [this.href = "/householdexpenses/" + clientId;] or meant to [householdExpenseId]